.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["prettier", "@aerian/eslint-config"],
```

Comment:
please replace eslint config with @aerian/eslint-config-react
and add @aerian/eslint-config-typescript